core/embed/lib/gfx_color.h

```diff
@@ -22,8 +22,11 @@
 
 #include <stdint.h>
 
+#ifdef UI_COLORS32BIT
```

Comment:
I would prefer to maintain consistency with the existing name `GFX_COLOR_32BIT` and suggest using either `UI_COLOR32BIT` (or `UI_COLOR_32BIT`) and the Rust feature `ui_color32bit` (or `ui_color_32bit`).

Reply:
renamed: https://github.com/trezor/trezor-firmware/commit/92df5bf66d3e6964ec166f0b3e8be1a7643f6c33